test/e2e/e2e.test.ts

```diff
@@ -396,6 +396,10 @@ describe('keyword preservation', () => {
     expect(yaml.tenant.friendly_name).to.equal('##TENANT_NAME##');
     expect(yaml.tenant.support_email).to.equal('support@##DOMAIN##');
     expect(yaml.tenant.support_url).to.equal('https://##DOMAIN##/support');
+    expect(
+      yaml.emailTemplates.find(({ template }) => template === 'welcome_email').resultUrl
+    ).to.equal('https://##DOMAIN##/welcome');
```

Comment:
This assertion in the E2E tests proves the changes work because the email templates do not have a `name` field and instead rely on the `template` field for uniqueness.